tests/Fixer/Basic/BracesFixerTest.php

```diff
@@ -4671,4 +4672,245 @@ public function testDoWhileLoopInsideAnIfWithoutBrackets($expected, $input = nul
     {
         $this->doTest($expected, $input);
     }
+
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     * @param null|array  $configuration
+     *
+     * @dataProvider provideBraceNextLineReturnTypeHint
+     * @requires PHP 7.0
+     */
+    public function testBraceNextLineReturnTypeHint($expected, $input = null, array $configuration = null)
+    {
+        if (null !== $configuration) {
+            $this->fixer->configure($configuration);
+        }
+
+        $this->doTest($expected, $input);
+    }
+
+    public function provideBraceNextLineReturnTypeHint()
+    {
+        return [
+            [
+                '<?php
+        function foo($bar): void
+        {
+        }',
+                null,
```

Comment:
Please replace `null` with actual input code that requires changes (i.e. with opening brace on same line as `void`). The result code will then be checked again to make sure it is not changed (like current test case).

If possible, add input code to other test cases that don't have one as well.